### PR TITLE
fix print format string

### PR DESCRIPTION
**Pull request by CaoShuFeng**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/client-go/rest/request.go [hunk 1]
```diff
@@ -731,7 +731,7 @@ func (r *Request) request(fn func(*http.Request, *http.Response)) error {
 					}
 				}
 
-				glog.V(4).Infof("Got a Retry-After %s response for attempt %d to %v", seconds, retries, url)
+				glog.V(4).Infof("Got a Retry-After %d response for attempt %d to %v", seconds, retries, url)
```
Comment (nikhita) [inline staging/src/k8s.io/client-go/rest/request.go]: :+1:

**Comment by nikhita** [comment]
/assign @sttts 
for approval

File: staging/src/k8s.io/client-go/rest/request.go (hunk 1 above)
Comment (sttts) [reply staging/src/k8s.io/client-go/rest/request.go]: %ds

Comment (CaoShuFeng) [reply staging/src/k8s.io/client-go/rest/request.go]: done.

**Pull request merged**